src/be/mcjava/model/CustomerOrder.java

```diff
@@ -0,0 +1,89 @@
+package be.mcjava.model;
+
+
+import java.util.List;
+
+/**
+ * Contains all data related to the order of a customer
+ */
+public class CustomerOrder {
+    private Long id;
+    private String name;
+    private String telephoneNumber;
+    private List<OrderItem> itemsToOrder;
+    private boolean finished;
+    
+    public CustomerOrder(String name, String telephoneNumber, List<OrderItem> itemsToOrder) {
+        this(null,name,telephoneNumber,itemsToOrder);
+    }
+    
+    public CustomerOrder(Long id, String name, String telephoneNumber, List<OrderItem> itemsToOrder) {
```

Comment:
4 parameters zijn te veel.
Gebruik een Builder.

Reply:
Switched to builder.